PWGHF/TableProducer/candidateSelectorDplusToPiKPi.cxx

```diff
@@ -71,12 +87,18 @@ struct HfCandidateSelectorDplusToPiKPi {
       labels[1 + aod::SelectionStep::RecoSkims] = "Skims selection";
       labels[1 + aod::SelectionStep::RecoTopol] = "Skims & Topological selections";
       labels[1 + aod::SelectionStep::RecoPID] = "Skims & Topological & PID selections";
+      labels[1 + aod::SelectionStep::RecoML] = "ML selection";
       static const AxisSpec axisSelections = {kNBinsSelections, 0.5, kNBinsSelections + 0.5, ""};
       registry.add("hSelections", "Selections;;#it{p}_{T} (GeV/#it{c})", {HistType::kTH2F, {axisSelections, {(std::vector<double>)binsPt, "#it{p}_{T} (GeV/#it{c})"}}});
       for (int iBin = 0; iBin < kNBinsSelections; ++iBin) {
         registry.get<TH2>(HIST("hSelections"))->GetXaxis()->SetBinLabel(iBin + 1, labels[iBin].data());
       }
     }
+
+    if (applyML) {
+      hfMLResponse.init(pTBinsML, cutsML, cutDirML, modelPathsML, true);
```

Comment:
Is the optimisation actually doing something for the BDT models?

Reply:
Actually I don't know, I had trouble finding its actual purpose. But it was a typo when modifying the code, I meant to set it to `false` by default.